packs/feat-effects/effect-meddling-ancestor.json

```diff
@@ -0,0 +1,92 @@
+{
+    "_id": "vkgmMBMd2e2BwiwJ",
+    "img": "systems/pf2e/icons/spells/chromatic-image.webp",
+    "name": "Effect: Meddling Ancestor",
```

Comment:
Can we use the nomenclature of the class feature here? "Ancestral Influence" as the name of the effect, and "Martial" as the "strike" influence

Reply:
I did use the nomenclature of the book for the effect name `Curse of Ancestral Meddling`, but we can change it easily. The label was just me going too quick and forgetting what they called it.